src/main/java/com/google/codeu/data/Message.java

```diff
@@ -41,6 +42,16 @@ public Message(UUID id, String user, String text, long timestamp) {
     this.timestamp = timestamp;
   }
 
+  public Message(UUID id, String user, String text, long timestamp, double score) {
+    this(id, user, text, timestamp);
```

Comment:
Just wanted to say I learnt something new as I didn't know you could do this for Java constructors! :)